data/Yashkapure06.json

```
@@ -0,0 +1,28 @@
+{
+  "name":"Yash Kapure",
+  "type":"Personal",
+  "bio":"As a full-stack web developer, I have a strong foundation in a range of technologies. My expertise spans from front-end development to back-end programming.One of my favorite technologies to work with is React.js, particularly within the MERN stack. I am currently expanding my skillset to include React Native 📱 for mobile development.",
+  "avatar":"https://avatars.githubusercontent.com/u/61585443?v=4",
```

Use short url for avatar.
```suggestion
  "avatar":"https://github.com/Yashkapure06.png",
```

---

Good suggestion, but this is not a blocker for the pull request